dispatch/route.go

```diff
@@ -45,7 +45,7 @@ type Route struct {
 
 	// Equality or regex matchers an alert has to fulfill to match
 	// this route.
-	Matchers types.Matchers
+	Matchers labels.Matchers
```

Comment:
The doc comment has to be updated now. "Equality or regex matchers…" → "Matchers…"